### PR TITLE
Aftershock: Monofilament Whip CBM

### PR DESCRIPTION
**Pull request by John-Candlebury**
#### Summary
Mods "Aftershock: New Monofilament Whip CBM"

#### Purpose of change
Classic cyberpunk weapon that easily fits as a mod bionic. Can deserve some extra content built around it later.

#### Describe the solution
In essence an implanted spear.

#### Testing

Loaded installed and used.

### CODE REVIEW

File: data/mods/Aftershock/items/cbms.json [hunk 1]
```diff
@@ -17,6 +17,15 @@
     "price": 100000,
     "difficulty": 5
   },
+  {
+    "id": "afs_bio_monowhip",
+    "copy-from": "bionic_general",
+    "type": "BIONIC_ITEM",
+    "name": { "str": "Monofilament Whip CBM" },
+    "description": "A carbon nanotube wire covertly installed in your right arm. When activated, it extends to its full length and can be used to strike enemies both at close and long range.",
```
Comment (Maleclypse) [inline data/mods/Aftershock/items/cbms.json:25]:
```suggestion
    "description": "A carbon nanotube wire covertly installed in your right arm.  When activated, it extends to its full length and can be used to strike enemies both at close and long range.",
```

File: data/mods/Aftershock/items/weapons.json [hunk 2]
```diff
@@ -416,5 +416,23 @@
...
+    "description": "A long filament of nested carbon nanotubes only a few microns in diameter.  In essence a very thin, very strong and very sharp blade that you throw at a moment's notice.",
+    "weight": "40 g",
+    "volume": "200 ml",
+    "price": 3500,
+    "price_postapoc": 500,
+    "material": [ "nanotubes" ],
+    "flags": [ "REACH_ATTACK", "REACH3", "NONCONDUCTIVE", "BIONIC_WEAPON", "UNBREAKABLE_MELEE", "TRADER_AVOID" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SPIN", "RAPID" ],
+    "cutting": 27,
+    "to_hit": { "grip": "bad", "length": "long", "surface": "every", "balance": "clumsy" },
```
Comment (Maleclypse) [inline data/mods/Aftershock/items/weapons.json:435]: I think you nailed the to_hit on the hit.  It's awkward but dangerous.

File: data/mods/Aftershock/player/bionics.json [hunk 3]
```diff
@@ -32,6 +32,16 @@
     "flags": [ "BIONIC_TOGGLED", "BIONIC_WEAPON", "NO_UNWIELD" ],
     "act_cost": "35 J"
   },
+  {
+    "type": "bionic",
+    "id": "afs_bio_monowhip",
+    "name": { "str": "Monofilament Whip" },
+    "description": "A carbon nanotube wire covertly installed in your right arm. When activated, it extends to its full length and can be used to strike enemies both at close and long range.",
```
Comment (Maleclypse) [inline data/mods/Aftershock/player/bionics.json:39]:
```suggestion
    "description": "A carbon nanotube wire covertly installed in your right arm.  When activated, it extends to its full length and can be used to strike enemies both at close and long range.",
```

**Review by Maleclypse — COMMENTED** [review]
I like it :)

File: data/mods/Aftershock/items/weapons.json (hunk 2 above)
Comment (John-Candlebury) [reply data/mods/Aftershock/items/weapons.json:435]: Yeah I think the techniques could be gated under efficiencies once theres support for that even.

**Pull request merged**